HPXMLtoOpenStudio/resources/hotwater_appliances.rb

```diff
@@ -150,20 +153,62 @@ def self.apply(model, runner, hpxml_header, hpxml_bldg, weather, spaces, hot_wat
       end
       if fridge_schedule.nil?
         fridge_unavailable_periods = Schedule.get_unavailable_periods(runner, fridge_col_name, unavailable_periods)
-        fridge_weekday_sch = refrigerator.weekday_fractions
-        fridge_weekend_sch = refrigerator.weekend_fractions
-        fridge_monthly_sch = refrigerator.monthly_multipliers
-        fridge_schedule_obj = MonthWeekdayWeekendSchedule.new(model, fridge_obj_name + ' schedule', fridge_weekday_sch, fridge_weekend_sch, fridge_monthly_sch, Constants.ScheduleTypeLimitsFraction, unavailable_periods: fridge_unavailable_periods)
-        fridge_design_level = fridge_schedule_obj.calc_design_level_from_daily_kwh(rf_annual_kwh / 365.0)
-        fridge_schedule = fridge_schedule_obj.schedule
+        if !refrigerator.constant_coeffcients.nil? && !refrigerator.temperature_coefficients.nil?
+          # FIXME: what about unavailable periods here?
+          arbitary_maxval = 0.06
+          design_level_kw = rf_annual_kwh / 365.0 * arbitary_maxval
+          fridge_design_level = UnitConversions.convert(design_level_kw, 'kW', 'W')
+
+          # FIXME: the schedule that we enter here is arbitary, since the actuator overrides it anyway?
+          fridge_schedule = OpenStudio::Model::ScheduleRuleset.new(model)
+          fridge_schedule.setName(fridge_obj_name + ' schedule')
+
+          tin_sensor = OpenStudio::Model::EnergyManagementSystemSensor.new(model, 'Zone Mean Air Temperature')
+          tin_sensor.setName('tin s')
+          tin_sensor.setKeyName(rf_space.thermalZone.get.name.to_s)
```

Comment:
Note that [refrigerators can be located in MF spaces](https://openstudio-hpxml.readthedocs.io/en/latest/workflow_inputs.html#hpxml-refrigerators), where we use a schedule temperature instead of a thermal zone, so we'll need to handle that case too.

It should just be a matter of attaching the sensor to the schedule value (like [this](https://github.com/NREL/OpenStudio-HPXML/blob/b0f18ace1033af6fd9c084965dbab72f36fee93d/HPXMLtoOpenStudio/resources/waterheater.rb#L829-L832)) instead of a thermal zone (like [this](https://github.com/NREL/OpenStudio-HPXML/blob/b0f18ace1033af6fd9c084965dbab72f36fee93d/HPXMLtoOpenStudio/resources/waterheater.rb#L854-L857)).

Reply:
I moved the following methods from the HPXMLtoOS measure to their own class in `util.rb`:
* `get_space_or_schedule_from_location`
* `get_space_temperature_schedule`
* `get_space_from_location`

Therefore, `hotwater_appliances.rb` can call `get_space_or_schedule_from_location`. If the method returns a space, the EMS sensor is attached to a thermal zone. If the method returns a schedule, the EMS sensor is attached to the schedule value.